GoogleDataTransport/GDTCORLibrary/Public/GDTCORPlatform.h

```diff
@@ -88,6 +88,30 @@ GDTCORNetworkType GDTCORNetworkTypeMessage(void);
  */
 GDTCORNetworkMobileSubtype GDTCORNetworkMobileSubTypeMessage(void);
 
+/** Writes the given object to the given fileURL and populates the given error if it fails.
+ *
+ * @param obj The object to encode.
+ * @param filePath The path to write the object to. Can be nil if you just need the data.
+ * @param error The error to populate if something goes wrong.
+ * @return The data of the archive. If error is nil, it's been written to disk.
+ */
+NSData *_Nullable GDTCOREncodeArchive(id<NSSecureCoding> obj,
```

Comment:
Adding to the public api should make this a minor version bump

Reply:
Updated in the spreadsheet. CHANGELOG and podspec updates will come after this.